src/test/apps.test.js

```diff
@@ -1035,34 +1039,30 @@ describe("App Health component", function () {
     expect(width).to.equal("20%");
   });
 
-  it("health bar for healthy tasks has correct content", function () {
-    var content = this.component.props.children[0].props["data-tip-content"];
-    expect(content).to.equal("healthy");
-  });
+  describe("tooltip", function () {
 
-  it("health bar for unhealthy tasks has correct content", function () {
-    var content = this.component.props.children[1].props["data-tip-content"];
-    expect(content).to.equal("unhealthy");
-  });
+    beforeEach(function () {
+      this.renderer = TestUtils.createRenderer();
+      this.renderer.render(
+        <AppHealthBarWithTooltipComponent
+          model={this.model}/>);
+      var component = this.renderer.getRenderOutput();
+      this.content = component._store.props.children.props["data-tip-content"];
+    });
 
-  it("health bar for running tasks has correct content", function () {
-    var content = this.component.props.children[2].props["data-tip-content"];
-    expect(content).to.equal("running");
-  });
+    it("Healthy tasks are reported correctly", function () {
+      // Regexp matching because the content is a string of html
+      expect(this.content).to.match(/2.*Healthy.*\(.*40.*%\).*/);
+    });
```

Comment:
:lipstick: on a  :pig2: 

:laughing: 

seriously though, good luck maintaining these ... let's get that damn DOM in our tests once and for all :stuck_out_tongue: